src/components/surveys/survey-design/question-edit/QuestionEditRhs.tsx

```diff
@@ -119,7 +119,7 @@ const QuestionEditRhs: FunctionComponent<QuestionEditProps> = ({step, onChange,
             <StyledLabel14 htmlFor="q_id">Question Identifier</StyledLabel14>
             <StyledSimpleTextInput
               sx={{display: 'block', width: '200px'}}
-              onChange={e => onChange({...step, identifier: e.target.value})}
+              disabled={true}
```

Comment:
Disabling it makes it grayed-out and a bit harder to read. Ideally, it would be nice to have it styled so that it looks "readonly" instead of "disabled". Disabled text or images to imply "not applicable" where the identifier is very directly applicable - it's how the answers are mapped to the questions.